lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -469,6 +474,100 @@ def create_encounter_status(redcap_record: dict) -> str:
     return mapper[standardized_status]
 
 
+def create_immunization(record: dict, patient_reference: dict) -> Optional[list]:
+    """ Returns a FHIR Immunization resource entry """
+    immunization_entries = []
```

Comment:
Good find. immunization records seems like a great approach here for storing the covid vaccine information for the participant.

Reply:
Ok, we'll need to make some related changes to the FHIR ETL to get these included in `individual.details`, and to shipping views to parse them out.